x-pack/solutions/chat/plugins/workchat-app/public/application/components/integrations/listing/integration_catalog_view.tsx

```diff
@@ -0,0 +1,134 @@
+
+interface IntegrationCardData {
+  title: string;
+  icon: IconType;
+  description: string;
+  disabled?: boolean;
+}
+
+export const IntegrationCatalogView: React.FC = () => {
+  const { navigateToWorkchatUrl } = useNavigation();
+
+  const integrationCards: Record<IntegrationType, IntegrationCardData> = {
+    [IntegrationType.index_source]: {
+      title: i18n.translate('workchatApp.integrations.listView.importIndexCard', {
+        defaultMessage: 'Import Index',
+      }),
+      icon: getIntegrationIcon(IntegrationType.index_source),
+      description: i18n.translate('workchatApp.integrations.listView.importIndexDescription', {
+        defaultMessage:
+          'Choose an existing index to connect and start using it in your workflows without re-importing your data',
+      }),
+      disabled: false,
+    },
+    [IntegrationType.external_server]: {
+      title: i18n.translate('workchatApp.integrations.listView.externalServerCard', {
+        defaultMessage: 'External Server',
+      }),
+      icon: getIntegrationIcon(IntegrationType.external_server) as IconType,
+      description: i18n.translate('workchatApp.integrations.listView.externalServerDescription', {
+        defaultMessage: 'Connect to external servers for data processing.',
+      }),
+      disabled: false,
+    },
+    [IntegrationType.salesforce]: {
+      title: i18n.translate('workchatApp.integrations.listView.salesforceCard', {
+        defaultMessage: 'Salesforce',
+      }),
+      icon: getIntegrationIcon(IntegrationType.salesforce) as IconType,
+      description: i18n.translate('workchatApp.integrations.listView.salesforceDescription', {
+        defaultMessage:
+          'Connect your Salesforce account to bring in customer records, case data, and account insights for use in workflows',
+      }),
+      disabled: false,
+    },
+  };
+
+  const { euiTheme } = useEuiTheme();
+  return (
+    <KibanaPageTemplate data-test-subj="integrationsCatalogPage">
+      <IntegrationListView tab={'catalog'} />
+      <EuiHorizontalRule margin="none" css={{ height: 2 }} />
+      <KibanaPageTemplate.Section>
+        <EuiText>
+          <strong>Available</strong>
+        </EuiText>
+        <EuiSpacer size="m" />
+        <EuiFlexGrid columns={4}>
+          {Object.entries(integrationCards).map(([type, cardData]) => (
+            <EuiFlexItem grow={4}>
+              <EuiCard
+                layout="horizontal"
+                icon={
+                  <div
+                    style={{
+                      backgroundColor: euiTheme.colors.backgroundBaseSubdued,
+                      borderRadius: '50%',
+                      width: '60px',
+                      height: '60px',
+                      display: 'flex',
+                      alignItems: 'center',
+                      justifyContent: 'center',
+                      marginLeft: '-10px',
+                      marginRight: '15px',
+                    }}
```

Comment:
NIT: for consistency and readability, I'd rather have us always using `emotion` for inline styling, as it is now the recommended way of doing styling in Kibana/EUI. (I didn't comment on each occurrence) 

```ts
import { css } from '@emotion/css';

const myClassName = css`
  backgroundColor: ${euiTheme.colors.backgroundBaseSubdued};
  borderRadius: '50%';
  ...
`

<div className={myClassName}>...</>
```

(unless you have a specific reason to use plain inline styling instead?)